app/controllers/return_controller.js

```diff
@@ -1,12 +1,17 @@
-var Charge = require('../models/charge.js')
-var StateModel = require('../models/state.js')
-var views = require('../utils/views.js')
-var CORRELATION_HEADER = require('../utils/correlation_header.js').CORRELATION_HEADER
-var withAnalyticsError = require('../utils/analytics.js').withAnalyticsError
+'use strict'
 
-var logger = require('winston')
+// npm dependencies
+const logger = require('winston')
 
-let CANCELABLE_STATES = [
+// local dependencies
```

Comment:
This is fine for now, but if we move to XO it will want `// Local dependencies`

Reply:
But TBF if we do change `xo --fix` would sort it out

Reply:
lol